src/py/flwr/superexec/deployment.py

```diff
@@ -74,9 +91,22 @@ def _create_run(
 
     @override
     def start_run(
-        self, fab_file: bytes, override_config: Dict[str, str]
+        self,
+        fab_file: bytes,
+        override_config: Dict[str, str],
+        config: Optional[Dict[str, str]],
```

Comment:
The config should not be passed on every run, but only once right after loading the executor plugin and before executing any runs. We should probably define a new `Executor` method for this.

Reply:
Good point, I'll update the PR